examples/multiple-choice/index.es

```diff
@@ -0,0 +1,77 @@
+Element `multiple-choice`
+
+(class extends HTMLElement {
+
+  initialize () {
+    this.context.current = 0
+
+    this.context.questions_and_answers =
+      [
+        {
+          question: 'what popular tool for developing the client side application should we use?',
+          answers: ['React', 'Ember.js', 'Vue', 'Angular', 'Bootstrap', 'jQuery'],
+          // answer: undefined // There is no answer hehe
+        },
+
+        {
+          question: 'What city is known as "The Big 🍎 Apple ?"',
+          answers: ['Seattle, WA', 'Skokie, IL', 'New York City'],
+          answer: 2
+        }
+      ]
+  }
+
+  onidle () {
+    const
+      previous = this.select
+        `footer button:first-of-type`
+
+    , next = this.select
+        `footer button:last-of-type`
+
+    , beginning =
+        this.context.current == 0
+
+    , end =
+        this.context.current ==
+          (this.context.questions_and_answers.length -1)
+
+    this.enable (beginning, previous)
+    this.enable (end, next)
+  }
+
+  // helpers
+  enable (predicate, element) {
+    predicate
```

Comment:
it should be an issue because the project should document it's own style conventions and why, is all. i'm not suggesting you change the style, just that you explain what the rules are and where they came from somewhere. i.e. just use the badge
![image](https://user-images.githubusercontent.com/185732/27776292-03055eaa-5f6f-11e7-83c8-56c08bd5a0e3.png)